services/agora/src/css/app.scss

```diff
@@ -1,6 +1,11 @@
 // app global css in SCSS form
+@font-face {
+    font-family: AlbertSans;
+    src: url("./fonts/Albert_Sans/AlbertSans-VariableFont_wght.ttf");
```

Comment:
By loading only this one, you don't load AlbertSans-Italic-VariableFont_wght.ttf so italic mode is not right I think (double-checking now)